src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -111,6 +110,7 @@ const defaultProps = {
         accountID: '',
     },
     transactionViolations: {},
+    allReportActions: {},
```

Comment:
afaik this is still being passed down to the component, right? Is it still necessary?

Reply:
the original comment also stripped down the prop being passed as a dependency, is this still a thing?

Reply:
@kacper-mikolajczak Would you be able to check this one out please?

Reply:
Ok in that case we can go ahead with merge, right?

Reply:
cc @mountiny @shubham1206agra  I'm a little worried that the above mentioned PR can introduce a performance regression in this scope - didn't have time to measure it yet, but since it eg. maps/reduces all of the keys it can become a bottleneck rather quickly (if not already). This is not related to this PR, but I wanted to let you know that we should test it first with this in mind as it directly impacts all of our core metrics.

Reply:
Lets keep an eye out for that one, also I have asked Adam to try to add more complex test scenario for the reassure test so we can have better integrated protection from regressing this method